### PR TITLE
publiccloud: Set openqa_ttl tag

### PR DESCRIPTION
**Pull request by cfconrad**
This tag can be used by pcw to automatically delete the instance.
See also: https://github.com/cfconrad/pcw/pull/71

- Verification run: https://openqa.suse.de/tests/3905329

And it is visible in pcw like: http://s.qa.suse.de/5766

### CODE REVIEW

**Comment by cfconrad** [comment]
@ldevulder maybe these tags are also interesting for your terraform suite.Without a tag like `openqa_created_by` the instance will not be monitored by pcw.

**Comment by cfconrad** [comment]
@pdostal FYI

File: lib/publiccloud/provider.pm [hunk 1]
```diff
@@ -355,6 +355,7 @@ sub terraform_apply {
         $cmd .= "-var 'region=" . $self->region . "' ";
         $cmd .= "-var 'name=" . $name . "' ";
         $cmd .= "-var 'project=" . $args{project} . "' " if $args{project};
+        $cmd .= sprintf(qq(-var 'tags={"openqa_ttl":"%d"}' ), get_var('MAX_JOB_TIME', 7200));
```
Comment (pdostal) [inline lib/publiccloud/provider.pm]: @cfconrad thank you for letting me know. So if I understand correctly from this line, the `openqa_ttl` tag is actually equal to `MAX_JOB_TIME` variable? I would maybe add some small extra gap between those - like 5 minutes for example.

Comment (cfconrad) [reply lib/publiccloud/provider.pm]: I thought about that as well, but we always boot a VM which already have a gap. But why not adding some more. I will add 5min

Comment (pdostal) [reply lib/publiccloud/provider.pm]: You're right! That should be enough. You can add or keep it as is ;)

**Comment by asmorodskyi** [comment]
LGTM , the only question which I have ... how it worked before ? :D

**Pull request merged**